app/controllers/medicaid/sign_and_submit_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Medicaid
+  class SignAndSubmitController < MedicaidStepsController
+    private
+
+    def step_params
```

Comment:
I think we do not need to define this here if the attrs we are defining are just on the medicaid app itself because the default implementation in the parent controller looks at the step attrs to determine what to pull

Reply:
I think this is required because we want to update it with the current time it hits the controller, not necessarily what's sent as a param in the request (this is also the pattern for SignAndSubmit)!